network-api/networkapi/donate/pagemodels/help_page.py

```diff
@@ -0,0 +1,34 @@
+from wagtail.admin.panels import FieldPanel
+from wagtail.fields import StreamField
+from wagtail.models import Page
+from wagtail_localize.fields import SynchronizedField, TranslatableField
+
+from networkapi.donate.models import BaseDonationPage
+from networkapi.wagtailpages.pagemodels.customblocks.base_fields import base_fields
+
+
+class DonateHelpPage(BaseDonationPage):
+    template = "donate/pages/help_page.html"
+
+    parent_page_types = ["DonateLandingPage"]
+
+    subpage_types: list = []
+
+    max_count = 1
+
+    body = StreamField(base_fields, blank=True, use_json_field=True)
+
+    content_panels = Page.content_panels + [
+        FieldPanel("body"),
+    ]
+
+    translatable_fields = [
+        # Promote tab fields
+        SynchronizedField("slug"),
+        TranslatableField("seo_title"),
+        SynchronizedField("show_in_menus"),
+        TranslatableField("search_description"),
+        SynchronizedField("search_image"),
+        # Content tab fields
+        SynchronizedField("body"),
```

Comment:
Is this correct? Do we want to show the same `body` for all translated pages? Shouldn't this be a `TranslatableField`?

Reply:
Hey @jhonatan-lopes! Great point, after taking a look at the rest of our codebase it appears that I put this here by mistake. All of the other "body" fields are set to be a `TranslatableField` like you mentioned. I have updated this file to do the same! 

Thanks for taking a look 👍